productize-the-model.adoc

```diff
@@ -22,12 +22,11 @@ work, and it gives operations centralized control and peace of mind.
 To get started, {{ECLIPSE_CHE_URL}}[access the
 CodeReady Workspaces instance], and log in using the username and
 password you’ve been assigned
-(e.g. `{{ USER_ID }}/{{ CHE_USER_PASSWORD }}`):
+(e.g. `{{ USER_ID }}/{{ CHE_USER_PASSWORD }}`):
 
 image::che-login.png[che-login]
 
-Once you log in, you’ll be placed on your personal dashboard. Click on
-the name of the pre-created workspace on the left, as shown below (the
+Upon successful logging into CodeReadyWorkspaces, select Dashboard -> enter {{USER_ID}}/{{USER_ID}}-workspace and open the file (the
```

Comment:
You can provide a screenshot to help the user here. I don't  think is right to call it a file.